experiments/context_gating/configs/experiment/context_encoder_concat.yaml

```diff
@@ -0,0 +1,16 @@
+# @package _global_
+group: context_encoder_concat
+
+encoder:
+  weights: '/home/mohan/git/CARL/carl/context_encoders/saved_models/AE/Pendulum/model.zip'
```

Comment:
- [ ] change to relative path

Reply:
also, when saving the model, can we recover the model without the knowledge of the input_dim, latent_dim and hidden_dim? is it saved somewhere as well?

Reply:
if not, can you saved it somehow?

Reply:
Done. Check 8ec875b8fab83265667ea7d55e1b66ca6e85e1e2